documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -43,7 +43,9 @@ Specify the audio output destination.
 [[change-user-password]]
 ===== Password
 
-The default user on Raspberry Pi OS is `pi` with the password `raspberry`. You can change that here. Read about other xref:using_linux.adoc#linux-users[users].
+You can change the 'default' user password.
```

Comment:
There's no longer a default password, so this should probably just say "You can change your user password." ?